src/components/services/Spa.vue

```diff
@@ -1,39 +1,226 @@
 <template>
   <div class="spa-home">
-    <h1>Spa Home</h1>
-    <h3 class="greetings">{{ msg }}</h3>
+
+    <div>
+      <CustomHeader title="My Concierge" subtitle="Spa" />
+    </div>
+
+    <div class="navbar-container">
+      <nav>
+        <ul>
+            <li><a href="#">Home</a></li>
+            <li class="sep">|</li>
+            <li><a href="#">Calendar</a></li>
+            <li class="sep">|</li>
+            <li><a href="#">Services</a></li>
+        </ul>
+      </nav>
+    </div>
+
+    
+    <div class="main-container">
+
+    <div class="col1">
+
+    <div class="form-container">
+      <div class="form-box">
+        <div class="title">
+         <h3>Make a Reservation</h3> 
+        </div>
+        <div class="row">
+          <input type="text">
+          <input type="text">
+        </div>
+
+        <div class="row">
+          <input type="text">
+          <input type="text">
+        </div>
+
+        <div class="row">
+            <button>Submit</button>
+        </div>  
+      </div>
+    </div>
+
+    
+    <div class="contact-container">
+      <div class="contact">
+        <h3>Contact the Spa: 555-222-5656</h3>
+      </div>
+    </div>
+
+    </div>
+    
+    <div class="service-container">
+        <div class="services">
+          <div class="title">
+            <h3>
+              Spa Services
+            </h3>
+          </div>
+
+          <div>
+            <li v-for="(item, index) in items">
+            {{ item.message }}
+          </li>
+          </div>
+
+          </div>
+    </div>
+    </div>
   </div>
 </template>
 
 <script>
+import CustomHeader from '../shared/CustomHeader.vue';
+import { ref } from 'vue'
+
+
 export default {
-  data() {
-      return {
-          msg: 'Work in Progress ...'
-      }
-  }
+  components: {
+    CustomHeader
+  },
+    data() {
+        return {
+          items: [
+          { message: 'Hot Stone Massage' },
+          { message: 'Sport Massage' },
+          { message: 'Swedish Massage' },
+          { message: 'Reflexology' },
+          { message: 'Facial' },
+          { message: 'Manicure' },
+          { message: 'Pedicure' }
+      ]        
+      };
+    },
 }
 </script>
   
 <style lang="scss">
   .spa-home {
+
```

Comment:
Please use nesting for the scss selectors. This improves readability and can help with issues of scope and specificity down the line. See the admin dashboard for an example of how to do this. But basically .spa-home should be the top-level selector and other selectors should be nested within it, nav-bar stuff should be nested within .nav-bar-container, which should be nested within .spa-home. etc.

It is also helpful to put the selectors in the order they appear in the template above. So .nav-bar-container would be above .main-container, etc.